public/course.html

```
@@ -1,233 +1,267 @@
+
+      <!-- Main content -->
+      <div class="flex-1 min-w-0">
+
+        <!-- Not joined CTA -->
+        <div id="join-cta"
+          class="hidden bg-gradient-to-br from-indigo-50 to-purple-50 border border-indigo-100 rounded-2xl p-8 text-center mb-6">
+          <p class="text-4xl mb-3">&#128274;</p>
+          <h3 class="font-bold text-slate-800 text-lg mb-2">Join to access session details</h3>
+          <p class="text-slate-500 text-sm mb-4">Session locations and descriptions are encrypted and only revealed to
+            participants.</p>
+          <button id="btn-join" onclick="joinActivity()"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)